### PR TITLE
Update spelling to en-US: "organise"->"organize"

### PR DESCRIPTION
**Pull request by timohaver**
organise -> organize 
except for the CSS ID _Organise_detail_view_layout_

https://developer.mozilla.org/en-us/docs/MDN/Guidelines/Writing_style_guide#spelling

### CODE REVIEW

**Comment by wbamberg** [comment]
@chrisdavidmills , I think we should probably update the heading ID in this case too, don't you? It seems like asking for trouble to let them diverge.

**Comment by chrisdavidmills** [comment]
@wbamberg yes, good call. And the IDs are normaized to all lower case in the final render in Yari, so I generally change it like that in the source too.

Since you've already checked this one out, I'll let you have the review credit and do the honours of merging ;-)

File: files/en-us/learn/server-side/django/admin_site/index.html [hunk 1]
```diff
@@ -241,9 +241,9 @@ <h3 id="Add_list_filters">Add list filters</h3>
 
 <p><img alt="Admin Site - BookInstance List Filters" src="admin_improved_bookinstance_list_filters.png"></p>
 
-<h3 id="Organise_detail_view_layout">Organise detail view layout</h3>
+<h3 id="Organise_detail_view_layout">Organize detail view layout</h3>
```
Comment (wbamberg) [inline files/en-us/learn/server-side/django/admin_site/index.html:244]:
```suggestion
<h3 id="organize_detail_view_layout">Organize detail view layout</h3>
```

**Review by wbamberg — APPROVED** [review]
Thanks @timohaver !

**Pull request merged**